google-cloud-clients/google-cloud-bigtable/src/main/java/com/google/cloud/bigtable/data/v2/BigtableDataSettings.java

```diff
@@ -97,6 +97,11 @@ public String getAppProfileId() {
     return getTypedStubSettings().readRowsSettings();
   }
 
+  /** Returns the object with the settings used for point reads via ReadRows. */
```

Comment:
line 235 it says readRow, here it is ReadRows.

Reply:
oh... I might have overlooked it. Thanks!!